whitepapers/business-value/glossary.md

```diff
@@ -1 +1,17 @@
-This glossary will serve as a reference guide to assist non-technical managers to map key terms, concepts, and acronyms related to open source in business environments.
+# The Glossary for you!
+This file will serve as a reference guide to assist non-technical managers to understand key terms, concepts, and acronyms related to open source in business environments.
+
+# The Acronyms
+TODO - Talk Openly, Develop Openly.
+OS - Open Source.
+OSPO(s) - Open Source Program Office(s).
+.Md - Mark Down File.
+PR - Pull Requests. 
+SC - Steering Committee.
+WG - Working Group. 
+CFP - Call for Proposal. 
+
+# Concepts 
+Pull Requests - This is a way of asking someone most likely a reviewer to look at your proposed changes and approve (merge) or decline. 
+Steering Committee - These are group of people that serve as a board of directors regulating and approving our activities here in TODO.
```

Comment:
define SC as a general concept used in open source communities, not specific to TODO. Might be worth it to include "Board" and similar names used in communities / open source governance models